src/main/resources/beam-template.conf

```diff
@@ -764,6 +764,9 @@ beam.debug {
 
     writeRealizedModeChoiceFile = "boolean | false"
     messageLogging = "boolean | false"
+    # the max of the next 2 values is taken for the initialization step
+    simulationStuckThresholdMin = "int | 999999"
+    initialiazationStuckThresholdMin = "int | 180"
```

Comment:
That's correct. Some big simulation may have a very long init step (when the time is 0) because of a big number of agents that need to be initialized. It may last an hour or more. At the init step the probability of getting stuck is very low.
That value `simulationStuckThresholdMin = "int | 999999"` means that this feature (exiting beam in case of it got stuck) is disabled.
On medium size simulation a valid delay time during simulation may be a few minutes (1-3). I think that if the user set `simulationStuckThresholdMin` to 30 mins it would be effectively exiting beam if it is stuck.
On the big simulations I think the user could set it to 60 minutes. Then `initialiazationStuckThresholdMin` works fine.